### PR TITLE
feat: dev config tool update (fix #298)

### LINKED ISSUE #298: npm install fails to install normally on nodejs stable version
## Version
3.7.2

## Development Environment
Mac, Chrome79

## Current Behavior
* npm install fails to install normally on nodejs stable version
    1. Error output when installing karma version 1.x.x
    2. Fixed in `package-lock.json` update, but outputs version compatibility warning.
    3. After all, to update karma, webpack also needs to be updated.
## Expected Behavior
* can run `npm install` without a problem.

### PR DESCRIPTION
**Pull request by jinwoo-kim-nhn**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

#### issue

* https://github.com/nhn/tui.image-editor/issues/298

* npm install fails to install normally on nodejs stable version
    1. Error output when installing karma version 1.x.x
    2. Fixed in package-lock.json update, but outputs version compatibility warning.
    3. After all, to update karma, webpack also needs to be updated.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by junghwan-park** [comment]
@jinwoo-kim-nhn 바꾸게된 이유와 변경 내역이 github issue로 PR에 연결되어 관리되면 좋을것 같습니다.

File: karma.conf.js [hunk 1]
```diff
@@ -121,36 +120,45 @@ module.exports = function(config) {
...
             externals: {
                 fabric: 'fabric'
             },
             module: {
-                preLoaders: [
+                rules: [
+                    {
+                        test: /\.js$/,
+                        include: /src/,
+                        exclude: /node_modules|bower_components/,
```
Comment (dongsik-yoo) [inline karma.conf.js]: Doest this use bower?

File: karma.conf.js [hunk 2]
```diff
@@ -121,36 +120,45 @@ module.exports = function(config) {
...
-                    {
+                        exclude: /node_modules|bower_components/,
+                        loader: 'babel-loader?cacheDirectory',
+                        options: {
+                            babelrc: true
+                        }
+                    }, {
                         test: /\.styl$/,
-                        loader: 'css-loader!stylus-loader?paths=src/css/'
+                        use: [
```
Comment (dongsik-yoo) [inline karma.conf.js]: A brief version is `use: ['css-loader', 'stylus-loader']`.

**Comment by junghwan-park** [comment]
fabric 설치 방법이 바뀐다면 readme에 설명도 바뀌어야 할 것 같습니다!

File: webpack.config.js [hunk 3]
```diff
@@ -48,33 +49,68 @@ module.exports = {
...
+        rules: [
             {
                 test: /\.js$/,
-                exclude: /(node_modules|bower_components)/,
-                loader: 'eslint-loader'
-            }
-        ],
-        loaders: [
-            {
+                exclude: /node_modules|bower_components/,
```
Comment (dongsik-yoo) [inline webpack.config.js]: bower?

File: karma.conf.js (hunk 2 above)
Comment (jinwoo-kim-nhn) [reply karma.conf.js]: Thank you.

File: karma.conf.js (hunk 1 above)
Comment (jinwoo-kim-nhn) [reply karma.conf.js]: I knew wrong usage, bower did not need to excloud. Thank you.

File: webpack.config.js (hunk 3 above)
Comment (jinwoo-kim-nhn) [reply webpack.config.js]: I knew wrong usage, bower did not need to excloud. OK

**Pull request merged**